packages/components/psammead-most-read/src/item.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { node, oneOf, shape, string } from 'prop-types';
+import styled from 'styled-components';
+import { getCanon, getPica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  getSerifLight,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+
+const MostReadWrapper = styled.div`
+  display: flex;
+
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 4px;
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 6px;
+  }
+`;
+
+const ItemWrapper = styled.div``;
+
+export const CountWrapper = styled.p`
```

Comment:
It'd be better to use a span here surrounding the number. The naming change is to follow the convention of highlighting the underlying HTML element of the styled component. 
```suggestion
export const StyledCountSpan = styled.span`
```